### ISSUE TITLE
Flavour text/missions about exotic (as well as genetically modified) organisms

### ISSUE THREAD

**Issue by opusforlife2** [endless-sky/endless-sky#10659]
### Problem Description

The human civilisation spans a whopping quarter galaxy in size. Going by simple chance, there should be loads and loads of exotic organisms found on livable planets. This is because unlike the real world, this space has been previously occupied by several different sapient species, so clearly evolution of life happened all over instead of just one planet.

All dogs on Earth are a result of [millennia of selective breeding](https://en.wikipedia.org/wiki/Domestication_of_the_dog) by humans. There are genetic modification experiments on plants as well as animals, some of which are viable and are in current use. It's hard to believe that humans at least, if not aliens, haven't messed around with the genetic code of several organisms, considering the fact that they did it to _themselves_ (Alphas).

Also, I just really want to know what kinds of pets all the different aliens keep, as well as interesting ones in different stellar systems.

### Related Issue Links

N/A

### Desired Solution

There should be far more talk of interesting plants and creatures found throughout the galaxy. Right now the exotic stuff is largely found in Ember Waste, and that's mostly limited to space-faring species. Also that dragon mission.

### Alternative Approaches

There could be missions to transport exotic organisms from one place to another. Or transport of Rangers to control escaped animals from zoos.

Research missions to investigate newly discovered species. Rescue missions to help people/populations escape from hordes of wild animals.

The possibilities are endless.

### Additional Context

There would also be different microbes, so another avenue open to talk about is new diseases that were discovered and battled as humans explored the galaxy. Maybe you could land on planets that were entirely wiped out by pandemics (too soon?).

Other topics: invasive species, mass extinctions, improved medicines, varied local diets, and did I mention pets?

**Comment by reticent-rem** [comment]
I completely agree -- the only real exotic animals in human space that I've noticed are 

- The dragons you've mentioned and quoijuu from the "Dangerous Games" missions
- The exotic lizard from the "Courier 4" mission (dozens of times larger than a komodo dragon!)
- The "ubergoats" on Big Sky
- A few vague wildlife references like the "little indigenous wildlife aside from thick-furred mammals and some migratory birds" found on Nifel
- Not in human space, but Greenview is home to "Hei Low Io species preservation foundation, and each of its four branches features a different internal climate showcasing life from distant worlds. An infographic proclaims it to be the largest collection in the galaxy of rare organisms sourced from private collections, rebuilt from genetic samples, or rescued before being exterminated after being deemed invasive on Hai worlds. It also mentions that certain exhibits have been improved in **some cases with species recently acquired from human space through secret trade deals**"

Maybe new exotic animals could be added to a cargo list and be included in special jobs or missions to deliver them to Greenview. That could also serve as a centralized reference of exotic animals and plants added.

Another idea could be a mission chain of acquiring exotic animals for a pirate warlord who's trying to build their own menagerie, which would give a less savory player a tour of some of the alien creatures to be found in human space.

Even more minor things could be that some animal-related idioms or phrases might have the animal replaced, like an offhand mention that a new hovercraft can reach 30 gruskpower instead of 500 horsepower, and the player later seeing a Grusk or reading about them.

I'll get the ball rolling and whip up a culture mission with some exotic animals over the next week or so.
